orchagent/mirrororch.cpp

```diff
@@ -727,6 +882,18 @@ bool MirrorOrch::activateSession(const string& name, MirrorEntry& session)
     }
 
     session.status = true;
+
+    if (!session.src_port.empty() && !session.direction.empty())
+    {
+        status = configurePortMirrorSession(name, session, true);
+        if (status == false)
+        {
+            SWSS_LOG_ERROR("Failed to activate port mirror session %s\n", name.c_str());
```

Comment:
port/lag in the log message.

Reply:
it is taken care in configurePortMirrorSession. I can add here too.  Port and LAG, LAG member port info is logged.
 SWSS_LOG_ERROR("Failed to configure mirror session port %s: %s, status %d, sessionId %x\n",
                                 port.m_alias.c_str(), p.m_alias.c_str(), status, sessionId);

Reply:
Have taken care of this